### DISCUSSION TITLE
Making diffusers documentation more accessible

### DISCUSSION THREAD

**Discussion by ariG23498** [huggingface/diffusers#8734] [General]
Hey folks!

Thank you for having such a thorough documentation for the library. I really enjoy going through the guieds and tutorials.

I wanted to create this thread to provide a non-technical POV. There are many people who might not know specific terms (inpainting, outpainting, etc.) or model names, but want to solve a particular problem.

Upon searching for a problem statement with diffusion models in a search engine, I am mostly taken to reddit or some other websits but not to the diffusers documentation page. I think there might be a possibility of having docs that show how to solve tasks or problem statements as they were without being too criptic with niche terms and model names.

What do you all think?

CC: @stevhliu

**Comment by stevhliu** [comment]
Hey, thanks for the feedback @ariG23498! What do you think about having a glossary type page with definitions for terms and models? Perhaps something similar to this [page](https://huggingface.co/docs/transformers/en/glossary) from Transformers?

**Reply by ariG23498** [reply]
That might be a good start. Thank you for listening to the suggestion 😄

**Reply by stevhliu** [reply]
Another potential option would be to do something like [hf.co/tasks](https://huggingface.co/tasks) but just for diffusion topics.

**Reply by asomoza** [reply]
Oh, the tasks are really a good idea. I think the best option is a mix of both: tasks with guides, and for each technical term, we can link to a glossary where the term is explained.
